docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -1361,11 +1637,12 @@ The ``queue.del`` command removes a queue from the Cortex.
 reindex
 -------
 
-The ``reindex`` command reindexes a given node property. This is an administrative command that is typically used when data model updates have been pushed to a Cortex and existing node properties must be migrated to the new model.
+The ``reindex`` command reindexes a given node property. This is an administrative command that is
```

Comment:
reindex doesn't actually do anything currently

```storm reindex --help
Executing query at 2023/01/23 23:56:25.848


    Use admin privileges to re index/normalize node properties.

    NOTE: Currently does nothing but is reserved for future use.
```

Reply:
Well wow.

Reply:
Reworded and added references to devops docs.